webapp/components/sidebar_header_dropdown.jsx

```diff
@@ -125,6 +126,11 @@ export default class SidebarHeaderDropdown extends React.Component {
         });
     }
 
+    switchTeam(teamUrl) {
+        this.setState({showDropdown: false});
```

Comment:
Need a `e.preventDefault()` in here somewhere or we're going to add `#` to the end of the URL